File: Cpp_solving/exercise_07.cpp
/*
    Given three values, determine and print a legend accordingly: 
    “They form a triangle” or “They do not form a triangle.”
*/

#include <iostream>
using namespace std;

// I create the resolving function.
void form_or_not_a_triangle(int value1, int value2, int value3){
    if ((value1 == value2 && value1 != value3) || 
        (value2 == value3 && value2 != value1) || 
        (value3 == value1 && value3 != value2))
    {
        cout << "They form a triangle!";
    }
    else cout << "They do not form a triangle";
}

// Showing if they form a triangle or not
int main () {
    form_or_not_a_triangle(2,3,2);
 return 0;
}

File: Python_solving/exercise_07.py
# Given three values, determine and print a legend accordingly: 
#  “They form a triangle” or “They do not form a triangle.”
# In this exercise I am going to use an isosceles triangle.

# I create the resolutive function
def form_or_not_a_triangle(side1,side2,side3):
    if ((side1 == side2 and side1 != side3) or 
        (side2 == side3 and side2 != side1) or 
        (side3 == side1 and side3 != side2)):
        print("They form a triangle!")
    else: print("They do not form a triangle")
# Show it
form_or_not_a_triangle(1,2,1)